### ISSUE TITLE
Android App

### ISSUE THREAD

**Issue by ThesiiNCey** [cinnyapp/cinny-desktop#206]
### Describe the problem

Is it possible that you make in Android App because i love cinny and i don't like Element

### Describe the solution you'd like

An APK File

**Comment by grandeljay** [comment]
> i love cinny and i don't like Element

Preach, brother

**Comment by DearFox** [comment]
Yes, we can install the site as an application through a modern browser such as Chrome or Firefox, but the problem is that notifications do not always work well and are delivered, which I consider quite important when using the messenger.
In addition, in the form of an application it would be possible to add more functions that are don't possible in site mode.  Moreover, if you do not have the Internet at the moment, the history of your messages is still with you if you need it.  Well, or sending a message offline so that it is sent as soon as you have Internet access.  It may not be very safe, but it is convenient and can be optional.

**Comment by cshaa** [comment]
Tauri supports building for Android and iOS now. This shouldn't be too hard to implement!

**Comment by u9000-Nine** [comment]
See [Builds for mobile · Issue #392 · cinnyapp/cinny-desktop](https://github.com/cinnyapp/cinny-desktop/issues/392)
